packages/react/src/TreeView/TreeView.tsx

```diff
@@ -337,6 +358,7 @@ export type TreeViewItemProps = {
   onExpandedChange?: (expanded: boolean) => void
   onSelect?: (event: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>) => void
   className?: string
+  dragHandle?: ReactElement
```

Comment:
I'm not sure we need to require folks to pass `dragHandle` if they're passing `dragAndDrop` on the parent.

If we have a tree view where some nodes are draggable and others are not, we could assume all nodes are draggable by default. Then, we could add a prop to `TreeView.Item` that allows people to disable drag-and-drop functionality.

Reply:
+1 to this one as well. I don't think we'll have a scenario where individual tree nodes would be prevented from being dragged and dropped?

Reply:
> I don't think we'll have a scenario where individual tree nodes would be prevented from being dragged and dropped?

@ericwbailey we _could_. If the user didn't have access to see an item, they wouldn't be able to drag it.